Prog/Equity_Compile_Feature_Data.sas

```diff
@@ -277,7 +339,7 @@ run;
 
```

Comment:
The **length** statement in the **data afford;** step is having no effect because the _indicator_ variable already exists in the input data set (**afford_&geosuf**). You should move the **length** statement to the **afford_wd12, afford_cl17,** and **afford_city** data steps where the _indicator_ variable is first created.

Reply:
moved